### ISSUE TITLE
SLF4J warning at boot

### ISSUE THREAD

**Issue by ninrod** [JetBrains/Exposed#472]
Hi, and thanks for this beautiful library

I used the logging configuration from the main readme.md, but I'm getting a warning from SLF4J as follows: 

```sh
backend ➜ java -jar build/libs/chess-0.0.1.jar 
SLF4J: Failed to load class "org.slf4j.impl.StaticLoggerBinder".
SLF4J: Defaulting to no-operation (NOP) logger implementation
SLF4J: See http://www.slf4j.org/codes.html#StaticLoggerBinder for further details.
SQL: SELECT usuario.login, usuario.firstname, usuario.lastname, usuario.description FROM usuario
1: first
2: second
3: third
backend ➜ 
```

is there any way to fix this? 

thanks in advance

**Comment by dkindler** [comment]
Yeah I got this issue when I added exposed to a project that was using Flyway. Adding the following to my `build.gradle` fixed it up for me:

```
dependencies {
    ...
    compile group: 'org.slf4j', name: 'slf4j-api', version: '1.7.25'
    compile group: 'org.slf4j', name: 'slf4j-simple', version: '1.7.25'
}
```

**Comment by mddanishansari** [comment]
I don't know what are the purpose of `slf4j-api` and `slf4j-simple` in Exposed but I'm using `0.34.1` version of Exposed and there was no `slf4j-simple` but only `slf4j-api` and I was still  getting warning as in the original issue description by @ninrod 

So I added `slf4j-nop` instead of `slf4j-simple` as suggested here http://www.slf4j.org/codes.html#StaticLoggerBinder (link is in the warning) and the warning was gone.

My point is, is `slf4j-simple` even required because `slf4j-nop` worked for me and is just 4 KB when the other one is 15 KB

_Note: I apologize if you find my point irritating 😂_

**Comment by Tapac** [comment]
If you'll use `slf4j-nop` you'll miss all Exposed log messages, please check different `slf4j` backends http://www.slf4j.org/manual.html#swapping

**Comment by mddanishansari** [comment]
@Tapac thanks for more info on this and clearing the doubt

**Comment by mddanishansari** [comment]
I noticed that there's only `slf4j-api` in Exposed and no other logging framework, so is this the reason for that warning?

**Comment by Tapac** [comment]
slf4j is a facade for different logging frameworks and developers should choose which one (log4j, logback, jcl, etc) to choose as all of them have different set of features and configurations.
